verticapy/learn/tools.py

```diff
@@ -405,7 +405,6 @@ def load_model(name: str, cursor=None, input_relation: str = "", test_relation:
                 int(parameters_dict["max_ntree"]),
                 int(parameters_dict["max_depth"]),
                 int(parameters_dict["nbins"]),
-                parameters_dict["objective"],
```

Comment:
I wonder why removing the objective info!

Reply:
See answer in the next questions.